### PR TITLE
Add Terraform for the relation embedder

### PR DESCRIPTION
**Pull request by alexwlchan**
Part of https://github.com/wellcomecollection/platform/issues/4830

### CODE REVIEW

**Comment by warrd** [comment]
I think we also need a change of the works id minter terraform to read the denormalised index rather than merged.

**Comment by alexwlchan** [comment]
> we also need a change of the works id minter terraform to read the denormalised index rather than merged

Done.

File: pipeline/terraform/stack/service_work_relation_embedder.tf [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    aws_security_group.service_egress.id,
+    aws_security_group.interservice.id,
+  ]
+
+  cluster_name = aws_ecs_cluster.cluster.name
+  cluster_arn  = aws_ecs_cluster.cluster.id
+
+  env_vars = {
+    metrics_namespace    = "${local.namespace_hyphen}_relation_embedder"
+    messages_bucket_name = aws_s3_bucket.messages.id
```
Comment (warrd) [inline pipeline/terraform/stack/service_work_relation_embedder.tf:27]: Not sure what this bucket is for, maybe copy / paste from an app that uses `big_messaging`? :thinking:

**Review by kenoir — COMMENTED** [review]
Needs an ECR repo provisioning?

**Comment by alexwlchan** [comment]
ECR repo came separately to unbreak the build.

**Review by kenoir — APPROVED** [review]

**Pull request merged**